lib/sass/script/value/map.rb

```diff
@@ -57,7 +57,7 @@ def to_sass(opts = {})
 
       to_sass = lambda do |value|
         if value.is_a?(Map) || (value.is_a?(List) && value.separator == :comma)
-          "(#{value.to_sass(opts)})"
+          "#{value.to_sass(opts)}"
```

Comment:
The parens here are still necessary for lists. Also, as the code stands currently, there's no reason to have this branch different than the one below.